ci/node.js.yml

```diff
@@ -21,4 +21,4 @@ jobs:
       run: |
         npm install
         npm run build --if-present
-        npm test
+        CI=true npm test

```

Comment:
I would set this as an env on the run step

```
      run: |
        npm install
        npm run build --if-present
        npm test
     env:
       CI: true
```

Reply:
Sure, thanks for the suggestion! https://github.com/actions/starter-workflows/pull/45/commits/c420664025b59150359485a6df349135a17847a9